webapp-django/crashstats/supersearch/views.py

```diff
@@ -97,6 +98,12 @@ def get_params(request):
     return params
 
 
+@ratelimit(
+    key='ip',
+    rate=utils.ratelimit_rate,
+    method=ratelimit.ALL,
+    block=True
+)
 @pass_default_context
 def search(request, default_context=None):
```

Comment:
This is the wrong view to limit, it is only loaded the initial page but not the results. You want to limit ``search_results`` instead, as that's the view that's calling Elasticsearch.